src/sentry/services/hybrid_cloud/organization_mapping/impl.py

```diff
@@ -77,17 +41,18 @@ def upsert(self, organization_id: int, update: RpcOrganizationMappingUpdate) ->
                 organization_id=organization_id, defaults=update_dict
             )
 
-    def verify_mappings(self, organization_id: int, slug: str) -> None:
-        try:
-            mapping = OrganizationMapping.objects.get(organization_id=organization_id, slug=slug)
-        except OrganizationMapping.DoesNotExist:
-            return
-
-        mapping.update(verified=True, idempotency_key="")
-
-        OrganizationMapping.objects.filter(
-            organization_id=organization_id, date_created__lte=mapping.date_created
-        ).exclude(slug=slug).delete()
+        org_slug_reservation_qs = OrganizationSlugReservation.objects.filter(
+            organization_id=organization_id
+        )
+        if not org_slug_reservation_qs.exists():
+            OrganizationSlugReservation(
+                region_name=update.region_name,
+                slug=update.slug,
+                organization_id=organization_id,
+                user_id=0,
```

Comment:
I needed a default user ID to put here, but it might be worth making this a constant (if there isn't one already)

Reply:
Hmmmmm... I'm trying to understand this particular line I suppose.  Something seems fishy.